external-crates/move/crates/move-binary-format/src/normalized.rs

```diff
@@ -193,6 +227,15 @@ pub enum Bytecode {
     CastU16,
     CastU32,
     CastU256,
+    PackVariant(VariantHandle),
+    PackVariantGeneric(VariantHandle),
+    UnpackVariant(VariantHandle),
+    UnpackVariantImmRef(VariantHandle),
+    UnpackVariantMutRef(VariantHandle),
+    UnpackVariantGeneric(VariantHandle),
+    UnpackVariantGenericImmRef(VariantHandle),
+    UnpackVariantGenericMutRef(VariantHandle),
+    VariantSwitch(VariantJumpTable),
```

Comment:
I was scrolling quickly and was like wtf... then realized this was `normalized.rs`... proceed :)